src/components/CommentSection.jsx

```diff
@@ -28,6 +30,38 @@ function CommentSection({ articleId, commentCount }) {
         );
     });
 
+    function handleCommentBodyChange(e) {
+        setNewCommentBody(e.target.value);
+    }
+
+    function handleSubmit(e) {
+        e.preventDefault();
+        setIsError(() => false);
+        if (newCommentBody === "") {
+            setIsError(() => {
+                return { msg: "Please write a comment before submitting" };
+            });
```

Comment:
Good point, I started off with true/false, but switched to objects later on in case I want to add anything more info to errors!

I should update the errors across all my code from "isError" to "error" to reflect that as well.